docs/csharp/language-reference/operators/snippets/shared/operators.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net10.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

This should be .net10, as this area uses preview features:

```suggestion
    <TargetFramework>net10.0</TargetFramework>
```